modules/ossm-deploy-cluster-wide-control-plane-console.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-create-smcp.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-deploy-cluster-wide-control-plane-console_{context}"]
+= Configuring the control plane for cluster-wide deployment with the web console
+
+You can configure the `ServiceMeshControlPlane` resource for cluster-wide deployment using the {product-title} web console.  In this example, `istio-system` is the name of the {SMProductShortName} control plane project.
+
+.Prerequisites
+
+* The {SMProductName} Operator must be installed.
+* An account with the `cluster-admin` role.
+
+.Procedure
+
+. Log in to the {product-title} web console as a user with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
+
+. Create a project named `istio-system`.
++
+.. Navigate to *Home* -> *Projects*.
++
+.. Click *Create Project*.
++
+.. In the *Name* field, enter `istio-system`. The `ServiceMeshControlPlane` resource must be installed in a project that is separate from your microservices and Operators.
++
+These steps use `istio-system` as an example, but you can deploy your {SMProductShortName} control plane in any project as long as it is separate from the project that contains your services.
++
+.. Click *Create*.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. Click the {SMProductName} Operator, then click *Istio Service Mesh Control Plane*.
+
+. On the *Istio Service Mesh Control Plane* tab, click *Create ServiceMeshControlPlane*.
+
+. Click *YAML file*.
+
+. Create a `ServiceMeshControlPlane` file named `istio-installation.yaml` using the following example. The version of the {SMProductShortName} control plane determines the features available regardless of the version of the Operator.
```

Comment:
No need to create a file when creating the SMCP via the web console. When you click "YAML view", you are presented with an example YAML that you must then modify/replace.

Reply:
@luksa I deleted this step and tweaked the one before and after it.